src/platform/plugins/shared/dashboard/public/dashboard_client/dashboard_client.ts

```diff
@@ -89,6 +95,7 @@ export const dashboardClient = {
     });
   },
   update: async (id: string, dashboardState: DashboardState, references: Reference[]) => {
+    const { access_control: accessControl, ...restOfData } = dashboardState;
```

Comment:
This should happen in update REST endpoint, src/platform/plugins/shared/dashboard/server/api/update/update.ts, before calling `transformDashboardIn`, not in client code